editor/src/clj/editor/gui.clj

```diff
@@ -1067,9 +1068,11 @@
   (output gpu-texture TextureLifecycle (g/fnk [texture-gpu-textures texture]
                                          (or (texture-gpu-textures texture)
                                              (texture-gpu-textures ""))))
-  (output texture-size g/Any (g/fnk [anim-data]
-                                    (when (some? anim-data)
-                                      [(double (:width anim-data)) (double (:height anim-data)) 0.0])))
+  (output size types/Vec3 (g/fnk [manual-size size-mode texture texture-size]
+                            (if (or (= :size-mode-manual size-mode)
+                                    (coll/empty? texture))
+                              manual-size
+                              texture-size)))
```

Comment:
The previous `texture-size` output is now a read-only property so that it shows in the Property Editor.

Meanwhile, we've made the `manual-size` property that was previously hidden visible, and converted the `size` property (that used to represent both manual and texture size) into an output. This makes it more clear what is going on when the size-related properties are overridden.